syncopy/tests/test_specest.py

```diff
@@ -722,7 +727,8 @@ def test_tf_toi(self):
         cfg.toi = "all"
         cfg.t_ftimwin = 0.05
         tfSpec = freqanalysis(cfg, self.tfData)
-        assert tfSpec.taper.size > 1
+        print(self.tfData)
```

Comment:
debug print (this might've been me, actually...)

Reply:
can remove it in any case!